Navigate Error

<!-- 不具合のテンプレート -->
**概要**
```
[fleet_adapter-15] terminate called after throwing an instance of 'pybind11::error_already_set'
[fleet_adapter-15]  what(): TypeError: '>' not supported between instances of 'NoneType' and 'float'
[fleet_adapter-15]
[fleet_adapter-15] At:
[fleet_adapter-15]  /home/todadx/rmf_ws/install/fleet_adapter_kachaka/lib/python3.10/site-packages/fleet_adapter_kachaka/RobotClientAPI.py(111): navigate
[fleet_adapter-15]  /home/todadx/rmf_ws/install/fleet_adapter_kachaka/lib/python3.10/site-packages/fleet_adapter_kachaka/fleet_adapter.py(240): navigate
[fleet_adapter-15]  /home/todadx/rmf_ws/install/fleet_adapter_kachaka/lib/python3.10/site-packages/fleet_adapter_kachaka/fleet_adapter.py(221): <lambda>
[fleet_adapter-15]
[ERROR] [fleet_adapter-15]: process has died [pid 176690, exit code -6, cmd '/home/todadx/rmf_ws/install/fleet_adapter_kachaka/lib/fleet_adapter_kachaka/fleet_adapter --config_file /home/todadx/rmf_ws/install/fleet_adapter_kachaka/share/fleet_adapter_kachaka/config0.yaml --nav_graph /home/todadx/rmf_ws/install/rmf_demos_maps/share/rmf_demos_maps/maps/tfit5fdx/nav_graphs/0.yaml --ros-args --params-file /tmp/launch_params_spkarh4n'].
```


**再現手順**
**修正しないとどう困るか**
**原因**

speed_limitがNoneが入ってくるときにErrorが起きる。

**修正案**

--- a/fleet_adapter_kachaka/fleet_adapter_kachaka/RobotClientAPI.py
+++ b/fleet_adapter_kachaka/fleet_adapter_kachaka/RobotClientAPI.py
@@ -29,6 +29,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Optional
+
 import requests
 
 
@@ -94,7 +96,7 @@ class RobotAPI:
         except requests.RequestException:
             return False
 
-    def navigate(self, robot_name: str, pose: list[float], map_name: str, speed_limit: float = 0.0) -> bool:
+    def navigate(self, robot_name: str, pose: list[float], map_name: str, speed_limit: Optional[float] = None) -> bool:
         """
         Request the robot to navigate to the specified pose.
 

--- a/fleet_adapter_kachaka/fleet_adapter_kachaka/RobotClientAPI.py
+++ b/fleet_adapter_kachaka/fleet_adapter_kachaka/RobotClientAPI.py
@@ -108,11 +108,12 @@ class RobotAPI:
             bool: True if the navigation request is successful, False otherwise.
         """
         # Set linear velocity based on speed limit
-        linear_velocity = speed_limit if speed_limit > 0.0 else 1.0
-        velocity = {"linear": linear_velocity, "angular": 1.0}
+        if speed_limit is not None:
+            linear_velocity = speed_limit if speed_limit > 0.0 else 1.0
+            velocity = {"linear": linear_velocity, "angular": 1.0}
 
-        url = self.prefix + "kachaka/set_robot_velocity"
-        requests.post(url, json=velocity)
+            url = self.prefix + "kachaka/set_robot_velocity"
+            requests.post(url, json=velocity)
 
         url = self.prefix + "kachaka/move_to_pose"
         position = {"x": pose[0], "y": pose[1], "yaw": pose[2]}

